src/composer.ts

```diff
@@ -55,7 +55,7 @@ type GuardedContext<
  * that narrows down `tt.Update` when intersected with it.
  */
 type MountMap = {
-  [T in tt.UpdateType]: Record<T, object>
+  [T in tt.UpdateType]: Record<T, unknown>
```

Comment:
Less accurate and less readable. Any reason to do this?

Reply:
Aesthetics. `T & unknown` is diplayed as `T`, leaving more space for all the union variants.